APP-MANAGER

```diff
@@ -44,24 +48,6 @@ function _no_sudo() {
 	fi
 }
 
-_no_sudo
```

Comment:
if you can, try to leave the function as one of the first and standalone, so whatever the user want to do, him can never use sudo in the command.

As usual, many people still use sudo for everything. This function, if starts at the really begin, prevents this syntax.

Reply:
I am not started yet...